app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -524,6 +524,8 @@ describe("the new project page", () => {
     });
     cy.findByRole("button", { name: /^submit/i }).click();
     cy.url().should("include", "/cif/projects");
+    // this checks that the project view list shows the milestone report status vs. the other report statuses
+    cy.findAllByRole("status").should("have.text", "Late");
```

Comment:
It would be good to have a dev chat about how to best test this bug fix (and generally, how to test queries) in our next promyze section. For now I added this assertion since it's just the one line cc @pbastia

Reply:
Ideally the bugfix would be tested in our integration tests (jest) instead of cypress

Reply:
https://github.com/bcgov/cas-cif/issues/827